operations/validate_csm_health.md

```diff
@@ -408,54 +166,45 @@ Known failures and how to recover:
 
   - The `spire-agent` service may also fail if an NCN was powered off for too long and its tokens expired. If this happens, delete `/root/spire/agent_svid.der`, `/root/spire/bundle.der`, and `/root/spire/data/svid.key` off the NCN before deleting the `request-ncn-join-token` daemonset pod.
 
-<a name="net-vault"></a>
-### 1.7 Verify the Vault Cluster is Healthy
-
-Execute the following commands on `ncn-m002`:
-
-```bash
-ncn-m002# goss -g /opt/cray/tests/install/ncn/tests/goss-k8s-vault-cluster-health.yaml validate
-```
-
-Check the output to verify no failures are reported:
-```
-Count: 2, Failed: 0, Skipped: 0
-```
-
-<a name="automated-goss-testing"></a>
-### 1.8 Automated Goss Testing
-
-There are multiple [Goss](https://github.com/aelsabbahy/goss) test suites available that cover a variety of sub-systems.
-
-Run the NCN health checks against the three different types of nodes with the following commands:
 
-**IMPORTANT:** These tests may only be successful while booted into the PIT node. Do not run these as part of upgrade testing. This includes the Kubernetes check in the next block.
+<a name="pet-optional-ncnhealthchecks-resources"></a>
+### 1.2 OPTIONAL Check of ncnHealthChecks Resources
 
+To dump the ncn uptimes, the node resource consumptions and/or the list of pods not in a running state, run the following:
 
 ```bash
-pit# /opt/cray/tests/install/ncn/automated/ncn-healthcheck-master
-pit# /opt/cray/tests/install/ncn/automated/ncn-healthcheck-worker
-pit# /opt/cray/tests/install/ncn/automated/ncn-healthcheck-storage
+ncn# /opt/cray/platform-utils/ncnHealthCheck.sh -s ncn_uptimes
+ncn# /opt/cray/platform-utils/ncnHealthCheck.sh -s node_resource_consumption
+ncn# /opt/cray/platform-utils/ncnHealthCheck.sh -s pods_not_running
 ```
+<a name="known-issues"></a>
+#### 1.2.1 Known Issues
 
-And the Kubernetes test suite via:
+* pods_not_running
+  - If the output of `pods_not_running` indicates that there are pods in the `Evicted` state, it may be due to the /root file system being filled up on the Kubernetes node in question. Kubernetes will begin evicting pods once the root file system space is at 85% until it is back under 80%. This may commonly happen on ncn-m001 as it is a location that install and doc files may be downloaded to. It may be necessary to clean up space in the /root directory if this is the root cause of pod evictions. The following commands can be used to determine if analysis of files under /root is needed to free-up space.
 
-```bash
-pit# /opt/cray/tests/install/ncn/automated/ncn-kubernetes-checks
-```
+    ```bash
+    ncn# df -h /root
+    Filesystem      Size  Used Avail Use% Mounted on
+    LiveOS_rootfs   280G  245G   35G  88% /
+    ```
 
-<a name="autogoss-issues"></a>
-#### 1.8.1 Known Test Issues
+    ```bash
```

Comment:
There should be an explanation of what these commands are doing and why an admin needs to run it.

Reply:
done